app/src/main/java/com/bebetterprogrammer/tictactoe/activities/Iswin.kt

```diff
@@ -0,0 +1,61 @@
+package com.bebetterprogrammer.tictactoe.activities
+
+import com.bebetterprogrammer.tictactoe.R
+import kotlinx.android.synthetic.main.activity_gameplay.*
+
+class Iswin {
+    var winPosition = arrayOf(
+        arrayOf(0, 1, 2), arrayOf(3, 4, 5), arrayOf(6, 7, 8), arrayOf(0, 3, 6),
+        arrayOf(1, 4, 7), arrayOf(2, 5, 8), arrayOf(0, 4, 8), arrayOf(2, 4, 6)
+    )
+    var p1 = 0
+    var p2 = 0
+
+    fun iswin(ob: GamePlayActivity): Boolean {
```

Comment:
We should not pass the activity instance here, instead, we can pass the values, which are required to check the winner in this function. That way, this function is independent of activity and anyone gets the winner based on passed data. Also, this is just a logical class, so UI changes should not be done here.